python/pyarrow/__init__.py

```diff
@@ -105,6 +105,16 @@
 
 localfs = LocalFileSystem.get_instance()
 
+# Entry point for starting the plasma store
+
+def start_plasma_store():
```

Comment:
OK, sounds good. We could have something like `pyarrow.plasma.start` and `pyarrow.plasma.kill`